ui/page/components/account_selector.go

```diff
@@ -200,34 +203,44 @@ func (as *AccountSelector) Layout(gtx layout.Context) layout.Dimensions {
 }
 
 func (as *AccountSelector) ListenForTxNotifications(ctx context.Context) {
+	if as.TxAndBlockNotificationListener == nil {
+		as.TxAndBlockNotificationListener = listeners.NewTxAndBlockNotificationListener()
+	}
+
+	err := as.WL.MultiWallet.AddTxAndBlockNotificationListener(as.TxAndBlockNotificationListener, true, AccoutSelectorID)
+	if err != nil {
+		log.Errorf("Error Adding Tx notifications listener: %+v", err)
```

Comment:
Not sure what's going on with the casing here. Also, any specific reason you're using `%+v` rather than just `%v'?
```suggestion
		log.Errorf("Error adding tx notifications listener: %v", err)
```